nav2_msgs/package.xml

```diff
@@ -1,10 +1,12 @@
 <?xml version="1.0"?>
 <?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
 <package format="3">
-  <name>nav2_libs_msgs</name>
+  <name>nav2_msgs</name>
   <version>0.1.0</version>
-  <description>TODO</description>
+  <description>Messages and service files for the navigation2 stack</description>
+  <author>Michael Jeronimo, Steve Macenski</author>
```

Comment:
We should avoid use of the author tag. Potentially anyone who has ever contributed a patch is an author. I'd expect the list to keep growing and growing over time, with no clear rule about when someone should be removed, or who gets top billing.

IMHO, the best solution is to not use this tag. The authorship is available in the git history.

Reply:
Will do.